public/v1/apps/elasticsearch.json

```diff
@@ -0,0 +1,83 @@
+{
+    "captainVersion": 1,
+    "documentation": "Elasticsearch (Single and Cluster). Taken from docker.elastic.co/elasticsearch/elasticsearch",
+    "dockerCompose": {
+        "version": "3.3",
+        "services": {
+            "$$cap_appname": {
+                "image" : "docker.elastic.co/elasticsearch/elasticsearch:$$cap_elasticsearch_version",
+                "notExposeAsWebApp": "true",
+                "volumes": [
+                    "$$cap_appname-elasticsearch-data:/usr/share/elasticsearch/data"
+                ],
+                "restart": "always",
+                "environment": {
+                    "ES_JAVA_OPTS": "-Xms512m -Xmx512m",
+                    "cluster.name": "$$cap_elasticsearch_cluster_name",
+                    "http.port": "$$cap_container_port",
+                    "node.master": "$$cap_elasticsearch_node_master",
+                    "node.data": "$$cap_elasticsearch_node_data",
+                    "discovery.zen.minimum_master_nodes": "$$cap_elasticsearch_minimum_master_nodes",
+                    "discovery.zen.ping.unicast.hosts": "$$cap_elasticsearch_unicast_hosts"
+                },
+                "ports": [
+                    "$$cap_host_port:$$cap_container_port"
+                ]
+            }
+        }
+    },
+    "instructions": {
+        "start": "Elasticsearch search engine. Be sure to extend the virtual memory on all your hosts! (increase \"echo 'vm.max_map_count=262144' >> /etc/sysctl.conf\" and restart \"sysctl -p\")",
+        "end": "Elasticsearch is deployed as $$cap_appname. Goto %%cap_host_port/_cat/health to check the health of your cluster."
```

Comment:
- Add a note to `end` saying that the deploy is finished. Elastic Search is starting up. You might see 502 errors for the next 2 minutes.
- Replace `%%cap_host_port` with `<YOUR-APP-URL>`